tasks/enable_only_testing_repo_for_pt.yml

```diff
@@ -0,0 +1,21 @@
+  - name: include playbook for removing Percona repo
+    include: remove_percona_repository.yml
+
+  - name: Install percona repository package
+    apt:
+#     deb: https://repo.percona.com/apt/pool/testing/p/percona-release/percona-release_1.0-21.generic_all.deb
+      deb: https://repo.percona.com/apt/percona-release_latest.generic_all.deb
+    when: ansible_os_family == "Debian"
+
+  - name: add the Percona testing yum repos for CentOS
+    yum: name=https://repo.percona.com/yum/percona-release-latest.noarch.rpm
+    when: ansible_os_family == "RedHat"
+
+  - name: enable {{repo_name}} testing repo
```

Comment:
you could simply have one single task with enables only a specific repo (set as a variable), less code to deal with, just a suggestion.

Reply:
The same: updated to have `../tasks/enable_only_repo_for_pt.yml` that uses `{{repo_name}} {{ repo_type}}` variables